config/answers.sam-installer.yaml

```diff
@@ -3,7 +3,9 @@ certs:
   generate: true
   deploy: true
   group: foreman
-katello: true
+katello:
+  package_names:
+    - katello-sam
```

Comment:
@ehelms started on this https://github.com/Katello/puppet-katello/pull/57/files to add a param for packages that may need scl prefixing..